nifi-assembly/pom.xml

```diff
@@ -1030,6 +1065,19 @@ language governing permissions and limitations under the License. -->
                                                 </includes>
                                             </dependency>
                                         </mapping>
+                                        <mapping>
+                                            <!-- TODO: remove this mapping once minimum Java version is 11 -->
+                                            <directory>/opt/nifi/nifi-${project.version}/lib/java11</directory>
```

Comment:
@jtstorck i dont think we can do an absolute path.  Needs to be relative to the install like other dirs

Reply:
i see - this is in the rpm context - all good thanks